src/factories/request.js

```diff
@@ -19,10 +19,16 @@ class Credentials {
 class RequestFactory {
   constructor(config) {
     this.config = config
-
     this.storage = config.storage
   }
 
+  resetProps(instance) {
```

Comment:
@samblacklock Perhaps we should make this a `static` so it can't be called on an instance of the `RequestFactory` class?